File: apple/green/src/main/java/net/tjalp/peach/apple/green/command/StopCommand.java
package net.tjalp.peach.apple.green.command;

import net.minestom.server.MinecraftServer;
import net.minestom.server.command.builder.Command;

public class StopCommand extends Command {

    public StopCommand() {
        super("stop", "end");

        setDefaultExecutor((sender, context) -> {
            sender.sendMessage("Stopping server");

            MinecraftServer.stopCleanly();
        });
    }
}

File: apple/green/src/main/kotlin/net/tjalp/peach/apple/green/command/StopCommand.kt
package net.tjalp.peach.apple.green.command

import net.kyori.adventure.text.Component
import net.minestom.server.MinecraftServer
import net.minestom.server.command.builder.Command

class StopCommand : Command("stop", "end", "shutdown") {

    init {
        setDefaultExecutor { sender, _ ->
            sender.sendMessage(Component.text("Stopping server..."))

            MinecraftServer.stopCleanly()
        }
    }
}